### DISCUSSION TITLE
VEML7700 ambient light sensor micropython module

### DISCUSSION THREAD

**Discussion by octaprog7** [micropython/micropython#9126] [Show and tell]
Micropython module for VEML7700, high accuracy ambient light sensor from Vishay Semiconductors: https://github.com/octaprog7/veml7700

**Comment by scruss** [comment]
How does it compare to [palouf34/veml7700: lib for micropython for veml7700](https://github.com/palouf34/veml7700)?

**Reply by octaprog7** [reply]
These two libraries are used to communicate with Vishay's VEML7700 sensor, but their code is not the same!
Due to the fact that the micropython forum is set to read-only mode, I had to use github.
In my opinion, the forum was much better suited for this than the github.
